README.md

```diff
@@ -6,7 +6,7 @@ Meanwhile JavaScript is evolving, notably with the upcoming release of ES2015. T
 
 # Discussion
 
-We've started the process of bringing together various communities using this format to move it forward into the ES2015 era and beyond. Feel free to join us! We'll be discussing in the issue tracker and in `#esprima` on Freenode.
+We've started the process of bringing together various communities using this format to move it forward into the ES2015 era and beyond. Feel free to join us! We'll be discussing in the issue tracker.
```

Comment:
This text looks like needs more updating - "we've started the process [...] to move it forward into the ES2015 era and beyond" is few years late now.

Reply:
I just removed this section for now. It doesn't provide much info that isn't already covered elsewhere. We can always make more edits to the README later.